### PR TITLE
Change "obsolete" to "archived"

### PR DESCRIPTION
**Pull request by cahrens**
Related to OEP-14, #37.

Please review @cpennington @jibsheet @pmitros (I don't know how many people I actually need...).

### CODE REVIEW

File: oeps/oep-0002.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Specification
 Each repo in the Open edX system will include a file ``openedx.yaml``, with the
```
Comment (cahrens) [inline oeps/oep-0002.rst]: Is there something that enforces these ``openedx.yaml`` files conform to the specifications below? If so, I will need to update it.

Comment (cpennington) [inline oeps/oep-0002.rst]: Yes, https://github.com/edx/repo-tools/tree/master/edx_repo_tools/oep2/checks has all of the automatic checkers that we've written for oeps.

Comment (cahrens) [inline oeps/oep-0002.rst]: What calls check_owner currently?

Comment (cpennington) [inline oeps/oep-0002.rst]: https://build.testeng.edx.org/job/oep2-report/

Comment (cpennington) [inline oeps/oep-0002.rst]: (In particular, that job runs the `oep2` script that's in https://github.com/edx/repo-tools.

**Pull request merged**